client/app/components/SkipToContent/index.jsx

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react';
+import css from './SkipToContent.scss';
+
+export type Props = {
+  id?: string,
+};
+
+function SkipToContent(props: Props) {
+
+  return <a className={css.skipToContent} href={`#${props.id}`}>Skip to main content</a>;
```

Comment:
Since our app supports internationalization (i18n) we'll want to add "Skip to main content" as a key in `config/locales/en.yml` and translate it in other languages too. You can use Google translate here for now.

Then you can use 

```
import { I18n } from '../../libs/i18n';

// etc.

I18n.t('navigation.skip_to_main_content'); 

```